Swiftfin/Views/SettingsView/CustomizeViewsSettings/Components/Sections/CustomizeHomeSection.swift

```diff
@@ -0,0 +1,49 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Defaults
+import SwiftUI
+
+extension CustomizeViewsSettings {
+    struct CustomizeHomeSection: View {
+
+        @Default(.Customization.Home.showRecentlyAdded)
+        private var showRecentlyAdded
+        @Default(.Customization.Home.maxNextUp)
+        private var maxNextUp
+        @Default(.Customization.Home.enableRewatching)
+        private var enableRewatching
+
+        var body: some View {
+            Section {
+                Toggle(L10n.showRecentlyAdded, isOn: $showRecentlyAdded)
+                Toggle(L10n.nextUpRewatch, isOn: $enableRewatching)
+                BasicStepper(
+                    title: L10n.nextUpDays,
+                    value: $maxNextUp,
+                    range: 0 ... 999,
+                    step: 1
+                )
+                .valueFormatter { days in
+                    switch days {
+                    case 0:
+                        return L10n.disabled
+                    case 1:
+                        return "1 " + L10n.day
```

Comment:
These `day/days` strings should instead take the integer day as a parameter instead of manual concatenation.

Reply:
I've updated this. I added a 'dayLabel' as an extension on Int. Let me know if you want me to do that another way. Saw that's how seconds were being handled and I liked centralizing this but I'm also not sure how much additional usage we'll get from this. I can go either way.